pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -64,6 +66,28 @@ sub _check_for_kernel_version {
     elsif ( ( $environment eq 'virtuozzo' ) || ( $environment eq 'lxc' ) ) {
         $self->add_info_advice( 'text' => ['Kernel updates are not supported on this virtualization platform. Be sure to keep the host’s kernel up to date.'] );
     }
+    elsif ( (@kernel_update) && ( $kc_kernelversion ) ) {
+        if ( kcare_kernel_version("check") eq "New version available" ) {
+            $self->add_bad_advice(
+                'text' => [
+                    'Kernel patched with KernelCare, but out of date. running kernel: [_1], most recent kernel: [list_and,_2]',
+                    $kc_kernelversion,
+                    \@kernel_update,
+                ],
+                'suggestion' => ['This can be resolved either by running ’/usr/bin/kcarectl --update’ from the command line to begin an update of the KernelCare kernel version, or by running ’yum update’ from the command line and rebooting the system.'],
+            );
+        }
+        else {
+            $self->add_info_advice(
```

Comment:
@bk2204  @ColMelvin 

It looks like we not sending an iContact on INFO should be fine will all the current advisor modules.  


https://github.com/CpanelInc/addon_securityadvisor/pull/29
The above contains some proposed POD for better defining the ADVISE types.   I'll submit CPANEL-6053 later this weekend to adjust check_security_advice_changes to not send iContact unless there is a change in WARN or higher.

cc: @sporks5000